### PR TITLE
Updating logo for mawikimaison.miraheze.org

### PR DESCRIPTION
**Pull request by CnocBride**
Per request T2349.

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -5991,6 +5991,7 @@
 		'makeiteasyautoswiki' => "//$wmgUploadHostname/makeiteasyautoswiki/0/01/Miea.png",
 		'malaysiawiki' => "//$wmgUploadHostname/malaysiawiki/1/1d/Malaysian_flag_round135.png",
 		'marinebiodiversitymatrixwiki' => "//$wmgUploadHostname/marinebiodiversitymatrixwiki/3/34/IUCN_logo_transparent_WICKI_web.jpg",
+		'mawwikimaisonwiki' => "//$wmgUploadHostname/mawikimaisonwiki/3/33/Wikihouse.png
```
Comment (Reception123) [inline LocalSettings.php]: you forgot the ", at the end

Comment (CnocBride) [reply LocalSettings.php]: Cheers.

**Comment by Reception123** [comment]
@CnocBride the comma is still missing :(

**Comment by CnocBride** [comment]
There that should be it.

**Comment by Reception123** [comment]
@CnocBride just noticed, you added an extra **w** for mawikimaison

**Pull request merged**